inception-ui-kb/src/main/resources/META-INF/asciidoc/user-guide/annotation_fact-extraction.adoc

```diff
@@ -0,0 +1,90 @@
+enter _1988_ for this qualifier value, and click *√* to save this qualifier.
+
+image::factExtraction3.png[align="center"]
+
+One can click the pencil icon on the same line as this qualifer to edit and delete this qualifier.
+
+==== Extracting a fact in the annotation page
+Assuming the project contains documents for annotation, one can now switch to the annotation page
+and choose the fact layer:
+
+image::factExtraction4.png[align="center"]
+
+One can now mark the predicate mention of a fact. In the screenshot below, _enrolled in_ is selected
+. The right sidebar shows feature editors for the predicate, the subject, the object and the
+qualifiers. One can then choose _educated-at_ in the dropdown field of the predicate feature editor
+to link the mention of this predicate to the property in the knowledge base. The candidate list of
+this dropdown field is a list of properties in the knowledge base.
+
+image::factExtraction5.png[align="center"]
+
+To annotate the subject mention of a fact, one needs to click *<Click to activate>* in the subject
+feature editor. After this input field turns orange, one can now mark the subject mention _Obama_of
+a fact. One can then choose _Barack Obama_ in the dropdown field of the subject feature editor to
+link the mention of this subject to the instance in the knowledge base. The candidate list of this
+dropdown field is a list of instances in the knowledge base. In the main editor, there is an arrow
+from the label above the predicate to the label above the subject, with a name _subject_.
+
+image::factExtraction6.png[align="center"]
+
+Extracting the object mention is same as extracting the subject.
+
+If this fact is already saved in the knowledge base, the label in the predicate feature editor shows
+ *There is at least one statement in the KB which matches for this SPO*.
+
+To extract a qualifier, one needs to choose the qualifier name from the drop-down field in front of
+the button *add* in the qualifier feature editor. The list of candidates for the qualifier name is a
+ list of properties from the knowledge base. After clicking *add*, a mention input field and a
+ dropdown field appear to collect this qualifier value information. In the screenshot below, the
+ qualifier name is _start-date_.
+
+image::factExtraction7.png[align="center"]
+
+One can then click to activate the mention input, annotate the mention, and choose an instance from
+the dropdown field to link the value of this qualifier to the knowledge base. The list of candidates
+ for the qualifier value is a list of instances from the knowledge base.
+
+image::factExtraction8.png[align="center"]
+
+After annotating this qualifer, in the main editor, there is an arrow from the label above the
+predicate to the label above the qualifier value, with the qualifier name _start-date_.
+
+So, a fact (Barack Obama, educated-at, Harvard Law School, Start-date: 1988) with its mentions is
+extracted in this example.
```

Comment:
It is rather annotated/linked than extracted, right? In order to extract it, the statement/qualifiers would need to be actually added to the KB.